Sources/Apodini/Semantic Model Builder/OpenAPI/OpenAPIConfiguration.swift

```diff
@@ -5,38 +5,67 @@
 import Foundation
 @_implementationOnly import OpenAPIKit
 
-private let openAPIInfoTitle = "Apodini-App"
-private let openAPIInfoVersion = "1.0.0"
+/// Default values used for OpenAPI configuration if not explicitly specified by developer.
+public enum OpenAPIConfigurationDefaults {
+    /// Default specification output format.
+    public static let outputFormat: OpenAPIOutputFormat = .json
+    /// Default specification output endpoint.
+    public static let outputEndpoint: String = "openapi"
+    /// Default swagger-ui endpoint.
+    public static let swaggerUiEndpoint: String = "openapi-ui"
+}
 
+/// The enclosing storage entity for OpenAPI-related information.
+struct OpenAPIStorageValue {
+    var document: OpenAPI.Document?
+    var configuration: OpenAPIConfiguration
+}
+
+/// The storage key for OpenAPI-related information.
+struct OpenAPIStorageKey: StorageKey {
+    typealias Value = OpenAPIStorageValue
+}
+
+/// An enum specifying the output format of the OpenAPI specification document.
+public enum OpenAPIOutputFormat {
+    /// JSON format output.
+    case json
+    /// YAML format output.
+    case yaml
+}
 
 /// A configuration structure for manually setting OpenAPI information and output locations.
-struct OpenAPIConfiguration {
+public class OpenAPIConfiguration: Configuration {
     /// General OpenAPI information.
-    var info: OpenAPI.Document.Info = OpenAPI.Document.Info(title: openAPIInfoTitle, version: openAPIInfoVersion)
+    var title: String?
+    var version: String?
```

Comment:
I think this is somehow a duplicate of the main version that is defined in the `WebService`. I would suggest using the version defined there for now. We might want to move the configuration of the Version to the configuration DSL in the future but this is out of scope for this PR.

Reply:
As far as I understand, the `WebService.version` is only accessible via `APIVersionContextKey` and hence only during `export`. If we don't want to include `WebService.version` in the storage, this seems to be only way to access it, unless I miss something. 
Therefore, the current solution [sets the `version` documented in OAS from the `APIVersionContextKey`](https://github.com/Apodini/Apodini/blob/43128c6477b988380b865ccf6ca4e17c3669eb02/Sources/Apodini/Semantic%20Model%20Builder/OpenAPI/OpenAPIInterfaceExporter.swift#L35). If a custom version is provided via `OpenAPIConfiguration(..., version: "v.1.1")` this will be overwritten.

Would you rather want to exclude it entirely from the constructor of `OpenAPIConfiguration`, such that it will always be set via `APIVersionContextKey`? Also, would you like me to integrate `WebService.version` into the `app.storage`?